lnwire/neighbor_ack_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) 2016 Bitfury Group Limited
+// Distributed under the MIT software license, see the accompanying
+// file LICENSE or http://www.opensource.org/licenses/mit-license.php
+
+package lnwire
+
+import (
+       "bytes"
+       "testing"
+
+       "github.com/roasbeef/btcd/wire"
+)
+
+func TestNeighborAckMessageEncodeDecode(t *testing.T) {
```

Comment:
This test doesn't really exercise anything other than the the `Encode/Decode` methods don't return errors given a null-initialized `NeighborAckMessage` message. 

At the very least, the two messages active at the end of the test should be checked for deep quality.